opm/simulators/linalg/PropertyTree.cpp

```diff
@@ -23,8 +23,14 @@
 
 #include <boost/property_tree/json_parser.hpp>
 
-namespace Opm
-{
+#include <memory>
+#include <optional>
+#include <ostream>
+#include <string>
+
+#include <stddef.h>
```

Comment:
we should probably change this to cstddef and instance for std::size_t. but okay to do later.

Reply:
> we should probably change this to cstddef and instance for std::size_t. but okay to do later.

I'll make a follow-up PR for that.